Sources/Box/Box.swift

```diff
@@ -0,0 +1,363 @@
+import Foundation
+import SwiftUI
+
+extension Warp {
+    private static let boxCornerRadius = 8.0
+
+    /// Asset name of the `Box` tooltip image.
+    private static let toolTipImageName = "icon_box-info"
+
+    /// Box is a layout component to display information in addition to the main content of a page.
+    public struct Box: View, Hashable {
+        /// Preferred style of box.
+        let style: BoxStyle
+
+        /// Text that will be shown as box's heading.
+        let title: String?
+
+        /// Text that will be shown after title in the middle of the box.
+        let subtitle: String
+
+        public typealias ButtonConstructor = (title: String, action: () -> Void)
+
+        /// Tuple that will provide a title and an action for creating a link view below subtitle.
+        /// Passing `nil` will skip adding link view.
+        let linkProvider: ButtonConstructor?
+
+        /// Tuple that will provide a title and an action for creating a button view below link.
+        /// Passing `nil` will skip adding button view.
+        let buttonProvider: ButtonConstructor?
+
+        /// Object responsible for providing colors in different environments and variants.
+        let colorProvider: ColorProvider
+
+        public static func == (lhs: Box, rhs: Box) -> Bool {
+            lhs.style == rhs.style && lhs.title == rhs.title
+        }
+
+        public func hash(into hasher: inout Hasher) {
+            hasher.combine(style)
+            hasher.combine(title)
+        }
+
+        public init(
+            style: Warp.BoxStyle,
+            title: String?,
+            subtitle: String,
+            link: ButtonConstructor? = nil,
+            button: ButtonConstructor? = nil,
+            secondaryButton: ButtonConstructor? = nil,
```

Comment:
box only has one button, and maybe add the missing self so it looks the same

Reply:
Done.